ios/AppLovinMAXAdView.m

```diff
@@ -20,6 +20,8 @@ @interface AppLovinMAXAdView()<MAAdViewAdDelegate, MAAdRevenueDelegate>
 @property (nonatomic, copy, nullable) NSString *customData;
 @property (nonatomic, assign, readonly, getter=isAdaptiveBannerEnabled) BOOL adaptiveBannerEnabled;
 @property (nonatomic, assign, readonly, getter=isAutoRefresh) BOOL autoRefresh;
+@property (nonatomic, copy, nullable) NSDictionary *extraParameter;
+@property (nonatomic, copy, nullable) NSDictionary *localExtraParameter;
```

Comment:
Since a dictionary can contain multiple parameters, let's make it plural. It will also match what you have in [ios/AppLovinMAXAdViewManager.m](https://github.com/AppLovin/AppLovin-MAX-React-Native/pull/182/files#diff-b301112ba4a754bcff65bfbdfd48e893aa5f9bb75f47f2f41b124e0376a4e180).

```suggestion
@property (nonatomic, copy, nullable) NSDictionary *extraParameters;
@property (nonatomic, copy, nullable) NSDictionary *localExtraParameters;
```

Reply:
Updated.